### DISCUSSION TITLE
mixcr align preset not recognized

### DISCUSSION THREAD

**Discussion by GregoireDS** [milaboratory/mixcr#876] [Q&A] [answered]
Hi,

We've updated to the latest version of MIXCR. Since then, I'm struggling with an error with **_mixcr align_**.

Error concerns  _**--preset**_  

In older version I used : **_mixcr align -p rna-seq_** 
In the newer version, the vignette indicate to use **_mixcr align -p align_rna-seq_4.0_** 
However I get the following error : 

_mixcr.jar align
No preset with name "align_rna-seq_4.0"_

I've tried to use instead : "_default_" or "_rna-seq_" but it does'nt work either.

Any suggestion? 

Thanks

**Comment by mizraelson** [comment, marked as answer]
Hi,
Currently the following presets are available:
rnaseq-tcr-cdr3
rnaseq-tcr-full-length

rnaseq-bcr-cdr3
rnaseq-bcr-full-length

Presets for BCR also work for  joint BCR and TCR data. In the upcoming update later this week we will add:

rnaseq-cdr3
rnaseq-full-length

So the naming will be more straightforward for the cases when both tcr and bcr are present.

**Reply by GregoireDS** [reply]
Ok thanks a lot, this is more clear !
